components/ai_chat/core/browser/types.h

```diff
@@ -17,6 +19,19 @@ struct SearchQuerySummary {
   bool operator==(const SearchQuerySummary& other) const = default;
 };
 
+struct Tab {
+  std::string id;
+  std::string title;
+  url::Origin origin;
+
+  Tab(const std::string& id,
```

Comment:
nit: you can do this without the constructor:

```
Tab{
    .id = ...,
    .title = ...,
    .origin = ...,
}?
```

Reply:
https://github.com/brave/brave-core/pull/28103/commits/f201f2f38a7d689a7bda9a59b93eed8e4554dc51